.github/workflows/issues.yaml

```diff
@@ -0,0 +1,31 @@
+name: Sync Issues to JIRA
+
+on:
+  issues:
+    types: [opened, reopened, closed]
+
+jobs:
+  update:
+    name: Update Issue
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create JIRA ticket
+        env:
+           ISSUE_TITLE: ${{ github.event.issue.title }}
+           ISSUE_DESCRIPTION: ${{ github.event.issue.body }}
+        run: |              
+          if ${{ contains(github.event.*.labels.*.name, 'Bug') }}; then
+            type=bug
+          else
+            type=story
+          fi
+          data=$( jq -n \
+                  --arg title "$ISSUE_TITLE" \
+                  --arg url '${{ github.event.issue.html_url }}' \
+                  --arg submitter '${{ github.event.issue.user.login }}' \
+                  --arg body "" \
```

Comment:
I assume `body` represents the content of the ticket. Is there any way to fetch it? Should we care to pass it too?

Reply:
I found the following comment in the original [action](https://github.com/beliaev-maksim/github-to-jira-automation/blob/master/.github/workflows/issues_to_jira.yaml):
```
 # canonical doesn't encourage discussions in JIRA on pulic project issues.
          # thus, it is NOT recommended to put description in JIRA in order to push conversation to GitHub
          # if strongly required, then replace '--arg body ""' with the next line
          # --arg body "$ISSUE_DESCRIPTION" 
```
The link to the github issue will be present in the Jira ticket anyway, so it might good to follow this practice.